src/components/look/search-box/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { Input } from 'antd';
+import styled from 'styled-components';
+
+import { P } from '@src/component/atoms';
+
+export type SearchBoxProps = {
+  onSearch: (keyword: string) => void;
+};
+
+function SearchBox({ onSearch }: SearchBoxProps) {
+  const [keyword, setKeyword] = useState('');
+
+  const handleChange = (e) => {
+    const {
+      target: { value },
+    } = e;
+    setKeyword(value);
+  };
+
+  const handleSearchKeyDown = (e) => {
```

Comment:
```suggestion
  const handleKeyDown = (e) => {
```

onChange -> handleChange 이므로
onKeyDown -< handleKeyDown이 맞습니다!

Reply:
10d728c